src/engine/blocks.js

```diff
@@ -298,7 +298,7 @@ class Blocks {
             if (!block.fields[args.name]) return;
             if (args.name === 'VARIABLE') {
                 // Get variable name using the id in args.value.
-                const variable = optRuntime.getEditingTarget().lookupVariableById(args.value);
```

Comment:
Hmmm looking at line 304 below, it looks like the `args.value` is the variable id?